tools/spectrast/.shed.yml

```diff
@@ -0,0 +1,11 @@
+owner: galaxyp
```

Comment:
@jj-umn I'm wondering if this should be integrated into Galaxy as part of the datatypes .... automatic conversion ....

Reply:
@bgruening   Maybe.   Since these rely on naming conventions, I worry a bit about errors.
I'm also thinking of adding tools spectrast_create, spectrast_filter, and spectrast_search as well. 
This first tool is for DIA MS workflow DIA-Umpire -> SearchGui/PeptideShaker -> Bibliospec -> SpectraST and then to skyline.

Reply:
Ok, then we keep them here. Interesting ... Skyline in Galaxy or outside?